src/Microsoft.AspNetCore.SignalR.Redis/RedisHubLifetimeManager.cs

```diff
@@ -23,18 +24,18 @@ public class RedisHubLifetimeManager<THub> : HubLifetimeManager<THub>, IDisposab
         private readonly ConnectionList _connections = new ConnectionList();
         // TODO: Investigate "memory leak" entries never get removed
         private readonly ConcurrentDictionary<string, GroupData> _groups = new ConcurrentDictionary<string, GroupData>();
-        private readonly InvocationAdapterRegistry _registry;
         private readonly ConnectionMultiplexer _redisServerConnection;
         private readonly ISubscriber _bus;
         private readonly ILoggerFactory _loggerFactory;
         private readonly RedisOptions _options;
 
-        public RedisHubLifetimeManager(InvocationAdapterRegistry registry,
-                                       ILoggerFactory loggerFactory,
+        // TODO: What protocol should we use to publish to the bus?
+        private readonly IHubProtocol _protocol = new JsonHubProtocol();
```

Comment:
We still need some kind of conversion - if a client speaks protobuf we can't send them JSON.